src/theme/icons/index.ts

```diff
@@ -13,3 +13,4 @@ export { chevronDown } from './chevronDown';
 export { arrowUp } from './arrowUp';
 export { quoteSquare } from './quoteSquare';
 export { caretDown } from './caretDown';
+export { plus } from './plus';
```

Comment:
Don't forget to add to the `Icon.enums.ts` file. I'll refactor this in the TableHeadCell PR and move that enum into `icons` folder